shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/dependencyInjection/RepositoryDI.kt

```diff
@@ -82,6 +86,7 @@ class RepositoryDI : IRepositories, KoinComponent {
     override val schedules: ISchedulesRepository by inject()
     override val searchResults: ISearchResultRepository by inject()
     override val settings: ISettingsRepository by inject()
+    override val sentry: ISentryRepository by inject()
```

Comment:
`sentry` should be before `settings` here. (Android Studio has a handy "Edit > Sort Lines" menu option.)

Reply:
Oh cool, thanks for the tip!